src/inpar/4C_inpar_IO_runtime_output_thermo.cpp

```diff
@@ -23,42 +23,58 @@ namespace Inpar
+          Core::Utils::SectionSpecs sublist_IO_vtk_output_thermo{sublist_IO_vtk_output, "THERMO"};
 
-        // whether to write temperature state
-        Core::Utils::bool_parameter(
-            "TEMPERATURE", "No", "write temperature output", sublist_IO_output_thermo);
+          // whether to write vtk output for thermo
+          Core::Utils::bool_parameter(
+              "VTK_OUTPUT_THERMO", "No", "write thermo output", sublist_IO_vtk_output_thermo);
 
-        // whether to write heatflux state
-        Core::Utils::bool_parameter(
-            "HEATFLUX", "No", "write heatflux output", sublist_IO_output_thermo);
+          // whether to write temperature state
+          Core::Utils::bool_parameter(
+              "TEMPERATURE", "No", "write temperature output", sublist_IO_vtk_output_thermo);
 
-        // whether to write temperature gradient state
-        Core::Utils::bool_parameter(
-            "TEMPGRAD", "No", "write temperature gradient output", sublist_IO_output_thermo);
+          // whether to write heatflux state
+          Core::Utils::bool_parameter(
+              "HEATFLUX", "No", "write heatflux output", sublist_IO_vtk_output_thermo);
 
-        // whether to write energy state
-        Core::Utils::bool_parameter(
-            "ENERGY", "No", "write energy output", sublist_IO_output_thermo);
+          // whether to write temperature gradient state
+          Core::Utils::bool_parameter(
+              "TEMPGRAD", "No", "write temperature gradient output", sublist_IO_vtk_output_thermo);
 
-        // whether to write element owner
-        Core::Utils::bool_parameter(
-            "ELEMENT_OWNER", "No", "write element owner", sublist_IO_output_thermo);
+          // whether to write element owner
+          Core::Utils::bool_parameter(
+              "ELEMENT_OWNER", "No", "write element owner", sublist_IO_vtk_output_thermo);
 
-        // whether to write element GIDs
-        Core::Utils::bool_parameter(
-            "ELEMENT_GID", "No", "write 4C internal element GIDs", sublist_IO_output_thermo);
+          // whether to write element GIDs
+          Core::Utils::bool_parameter(
+              "ELEMENT_GID", "No", "write 4C internal element GIDs", sublist_IO_vtk_output_thermo);
 
-        // whether to write node GIDs
-        Core::Utils::bool_parameter(
-            "NODE_GID", "No", "write 4C internal node GIDs", sublist_IO_output_thermo);
+          // whether to write node GIDs
+          Core::Utils::bool_parameter(
+              "NODE_GID", "No", "write 4C internal node GIDs", sublist_IO_vtk_output_thermo);
 
-        sublist_IO_output_thermo.move_into_collection(list);
+          sublist_IO_vtk_output_thermo.move_into_collection(list);
+        }
+
+        // csv runtime output
+        {
+          Core::Utils::SectionSpecs sublist_IO_csv_output{sublist_IO, "RUNTIME CSV OUTPUT"};
+          Core::Utils::SectionSpecs sublist_IO_csv_output_thermo{sublist_IO_csv_output, "THERMO"};
```

Comment:
First, I apologize for the current state of these parameters, this is not fully migrated to the `InputSpecs` yet, although they are used internally. The way we gather these parameters is still rather convoluted. 

Second, I am not a big fan of adding another of these inverted dependencies: Why should we have a global `RUNTIME CSV OUTPUT` section which is then filled with `THERMO` specific information? From a modularization perspective, we actively try to avoid and refactor such global collections (see e.g. #306). Can you move the thermo-specific choice to one of the thermo sections?

Reply:
@sebproell Ah of course, I didn't really think about that while changing the input. I can move it to some of the thermo sections 👍.